client/client.go

```diff
@@ -351,6 +351,15 @@ func (c Client) TanglesThread(o message.TanglesArgs) (*muxrpc.ByteSource, error)
 	return src, nil
 }
 
+func (c Client) InviteCreate(o message.InviteCreateArgs) (string, error) {
```

Comment:
Would have preferred `(invite.Token, error)` but then it was creating a circular import and I'm not diving into that rn :see_no_evil: